### PR TITLE
Remove dependency on govuk_frontend_toolkit

### PR DESCRIPTION
**Pull request by vanitabarrett**
## What
Removes dependency on govuk_frontend_toolkit. 

## Why
It is part of the long-term GOV.UK Frontend strategy to move away from govuk_frontend_toolkit as it is deprecated, and instead use GOV.UK Frontend.

## Pages to test
https://govuk-collections-pr-1310.herokuapp.com/brexit
https://govuk-collections-pr-1310.herokuapp.com/education
https://govuk-collections-pr-1310.herokuapp.com/topic
https://govuk-collections-pr-1310.herokuapp.com/topic/benefits-credits
https://govuk-collections-pr-1310.herokuapp.com/topic/benefits-credits/child-benefit
https://govuk-collections-pr-1310.herokuapp.com/topic/benefits-credits/child-benefit/email-signup
https://govuk-collections-pr-1310.herokuapp.com/browse
https://govuk-collections-pr-1310.herokuapp.com/browse/business/business-tax
https://govuk-collections-pr-1310.herokuapp.com/government/organisations
https://govuk-collections-pr-1310.herokuapp.com/government/organisations/attorney-generals-office
https://govuk-collections-pr-1310.herokuapp.com/world/afghanistan
https://govuk-collections-pr-1310.herokuapp.com/government/organisations/prime-ministers-office-10-downing-street
https://govuk-collections-pr-1310.herokuapp.com/government/organisations/crown-prosecution-service
https://govuk-collections-pr-1310.herokuapp.com/government/organisations/forestry-commission

### CODE REVIEW

File: app/assets/stylesheets/views/_taxons.scss [hunk 1]
```diff
@@ -62,53 +59,52 @@
...
     .taxon-page__grid-item {
-      padding-bottom: $gutter;
+      padding-bottom: govuk-spacing(6);
 
       @include media(tablet) {
         @include box-sizing(border-box);
         display: inline-block;
         width: 50%;
-        min-height: (5 * $gutter);
+        min-height: (5 * govuk-spacing(6));
```
Comment (andysellick) [inline app/assets/stylesheets/views/_taxons.scss]: Maybe just write this out as `px` so it's more readable.

File: app/assets/stylesheets/views/_taxons.scss [hunk 2]
```diff
@@ -62,53 +59,52 @@
...
         vertical-align: top;
-        padding: 0 $gutter-half $gutter * 1.5 0;
+        padding: 0 govuk-spacing(3) govuk-spacing(6) * 1.5 0;
```
Comment (andysellick) [inline app/assets/stylesheets/views/_taxons.scss]: Maybe just write this out as `px` so it's more readable.

File: app/assets/stylesheets/views/_browse.scss [hunk 3]
```diff
@@ -1,35 +1,33 @@
...
       .section-pane {
-        @include media(tablet) {
+        @include govuk-media-query(tablet) {
           float: right;
           width: 30%;
           margin-right: 45%;
         }
 
-        @include media(desktop) {
+        @include govuk-media-query(desktop) {
```
Comment (andysellick) [inline app/assets/stylesheets/views/_browse.scss]: Just want to check - the documentation for these media queries has them written like this: `@include govuk-media-query($from: desktop)` - will the above still work?

File: app/assets/stylesheets/views/_organisations.scss [hunk 4]
```diff
@@ -85,6 +84,7 @@
 
   .filter-organisations-list__label {
     @include govuk-font(36, $weight: bold);
+    margin-right: govuk-spacing(2);
```
[the remark this answers is not in the document]
Comment (vanitabarrett) [reply app/assets/stylesheets/views/_organisations.scss]: Do you mean the "What's the latest from" text? It's just taking up the space it needs to take up, I believe. There's a `<br>` to make sure the "from" wraps onto a separate line. From what I remember, we just copied over the page from Whitehall as it was without re-thinking the design so it's probably up for debate.

[the remark this answers is not in the document]
Comment (andysellick) [reply app/assets/stylesheets/views/_organisations.scss]: I did, yes. Oh there's a `<br>` in there? I was hunting through all the CSS, didn't think to check the markup. Yeah, don't worry about this for now - I think we should get it redesigned at some point but not now.

File: app/assets/javascripts/application.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-//= require vendor/polyfills/bind
+//= require govuk-frontend/govuk/vendor/polyfills/Function/prototype/bind
```
Comment (alex-ju) [inline app/assets/javascripts/application.js]: 👍

**Comment by vanitabarrett** [comment]
@andysellick Agreed - I'll address the focus states in a new PR 🙂

**Pull request merged**